ui/components/ui/list-item/list-item.stories.js

```diff
@@ -79,14 +79,17 @@ export const SendComponent = (args) => (
 SendComponent.argTypes = {
   secondaryButtonText: {
     control: 'text',
-    defaultValue: 'Speed up',
   },
   cancelButtonText: {
     control: 'text',
-    defaultValue: 'Cancel',
   },
 };
 
+SendComponent.args = {
+  secondaryButtonText: 'Speed up',
+  cancelButtonText: 'Cancel',
+};
+
 export const PendingComponent = (args) => (
   <ListItem
     title={args.title}
```

Comment:
List item story works the same as before. Updating deprecated `defaultValue` syntax

### Before
<img width="1440" alt="Screenshot 2022-12-11 at 7 01 35 PM" src="https://user-images.githubusercontent.com/8112138/206959062-aa740e00-0d9e-4ddc-bf26-a8d8788aa288.png">

### After
<img width="1440" alt="Screenshot 2022-12-11 at 7 04 38 PM" src="https://user-images.githubusercontent.com/8112138/206959065-750b2f37-1cbb-4239-857b-f4d8f62f691e.png">